crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -274,13 +274,13 @@ class MotionPlanningServer : public rclcpp::Node
       // Store trajectories for service response
       trajopt_trajectories = path_plan_results->final_trajectories;
       crs_msgs::msg::ProcessMotionPlan resulting_process;
-      if (path_plan_results->final_start_end_trajectories.size() > 0)
+      if (path_plan_results->final_start_trajectory.points.size() > 0)
```

Comment:
Does the "final_" prefix mean that it should be executed after the process trajectory?

Reply:
Final means the trajectory to be used, whether it is from trajopt or ompl